docker/compose/mint/docker-compose.yaml

```diff
@@ -3,16 +3,16 @@
 # The mint / faucet can be accessed at 127.0.0.1:8000
 #
 # Place the following files into the same directory:
-# * mint.key (also known as the libra_root_key)
+# * mint.key (also known as the diem_root_key)
 # * waypoint.txt
 #
 # Additional notes:
-# * Images can be found at https://hub.docker.com/r/libra/mint/tags, obtain the latest tag and
+# * Images can be found at https://hub.docker.com/r/diem/mint/tags, obtain the latest tag and
 # update below.
 version: "3.8"
 services:
     mint:
-        image: libra/mint:master_f53da21f
+        image: diem/mint:master_f53da21f
```

Comment:
Cannot replace.

Reply:
@sausagee I've updated these, but tried to keep as much "diem" as possible. Let me know if they look fine now :)